pallets/pool-system/src/lib.rs

```diff
@@ -358,6 +358,11 @@ pub mod pallet {
 		#[pallet::constant]
 		type PoolDeposit: Get<Self::Balance>;
 
+		/// Enable fast runtime changes, overwriting the waiting time with a
+		/// fixed value. `None` value means the normal behaviour.
+		/// Used for testnets.
+		type FastChanges: Get<Option<Seconds>>;
```

Comment:
I do not think we should add this as a pallet type. can we not fix this in the runtime itself?

Reply:
Yes, I think so that by doing some macros I have in mind, we can avoid this and let decide the runtime but without boilerplate

Reply:
I would favor that over having test-based types in the core pallet here.

Reply:
Agree